### PR TITLE
Added an entry to /applications for Jack Sanity.

### PR DESCRIPTION
**Pull request by psychoticmeow**
I've recently written an application that lets me control Jack connections from a Node.js script, I've added it to the list of Control Applications.

### CODE REVIEW

File: applications/index.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ menu_item: true
     is a LADISH frontend.
   * [**Carla (KXStudio)**](http://kxstudio.sourceforge.net/Applications:Carla)
     is an audio plugin host.
+  * [**Jack Sanit**](https://github.com/psychoticmeow/jack-sanity)
```
Comment (mseeber) [inline applications/index.md]: isn't it called "Jack Sanity" you may want to correct that typo in the name of your own app ;)

**Comment by mseeber** [comment]
please rebase on the current master, so i can auto merge.

**Comment by psychoticmeow** [comment]
There we go, it's like it never happened.

**Comment by mseeber** [comment]
nice, thank you

**Pull request merged**